### PR TITLE
Create 2411MasqueradeRevival.MD

### PR DESCRIPTION
**Pull request by SJCoding0130**

### CODE REVIEW

File: _events/2411MasqueradeRevival.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: Moonlight Dance ～Masquerade of Conspiracy～(Reprint)
+jp_title: 月光群舞 ～陰謀のマスカレード～(復刻)
+news_link: https://live-a-hero.jp/info/10026
+banner_image:
```
Comment (rongjie-alter) [inline _events/2411MasqueradeRevival.md:5]: banner_info_MasqueradeRevival2411.jpg

File: _events/2411MasqueradeRevival.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+| 13 | Clear event free quest rank C  | 400 {% include item-icon.html id=190 %}    |
+| 14 | Clear event free quest rank B  | 600 {% include item-icon.html id=190 %}    |
+| 15 | Clear event free quest rank A  | 600 {% include item-icon.html id=190 %}    |
+| 16 | Clear event free quest rank S  | 800 {% include item-icon.html id=190 %}    |
+| 17 | Clear event free quest beginner  | 600 {% include item-icon.html id=190 %}    |
+| 18 | Clear event free quest expert  | 800 {% include item-icon.html id=190 %}    |
+
+## Event Shop
+
+{% include shop-table.html id=53 %}
```
Comment (rongjie-alter) [inline _events/2411MasqueradeRevival.md:135]: id=72, you can find that out in https://github.com/liveahero-wiki/liveahero-wiki.github.io/commit/ab75611754de23ea8cacfb43e971e08aa39c2552

**Review by rongjie-alter — COMMENTED** [review]
Thanks for the contribution! Just need to fix a few more things, then I will merge next morning

File: _events/2411MasqueradeRevival.md (hunk 1 above)
Comment (SJCoding0130) [reply _events/2411MasqueradeRevival.md]: ok

**Comment by SJCoding0130** [comment]
Welcome, in fact this is my first time trying to edit LAH wiki.
BTW the gacha rate I havent change anything yet

**Comment by rongjie-alter** [comment]
> Welcome, in fact this is my first time trying to edit LAH wiki.

Welcome! Also, if you have a Discord account, do join us in https://discord.gg/zpc7PCk . I coordinate most of the wiki project effort there.

> BTW the gacha rate I havent change anything yet

I think it is the same as last time?

**Pull request merged**